plugins/cache-portals-on-map.js

```diff
@@ -29,38 +29,37 @@ window.plugin.cachePortalDetailsOnMap.entityInject = function(data) {
   }
   data.callback(ents);
 };
-/****************************************************************************************/
-/* extend portal's GUID lookup to search in cached portals as well                      */
 
-window.plugin.cachePortalDetailsOnMap.findPortalGuidByCacheE6 = function(latE6, lngE6) {
-  let guid = window.plugin.cachePortalDetailsOnMap.findPortalGuidByPositionE6old(latE6, lngE6);
+// extend portal's GUID lookup to search in cached portals as well
+window.cachePortalDetailsOnMap.findPortalGuidByCacheE6 = function(latE6, lngE6) {
+  let guid = window.cachePortalDetailsOnMap.findPortalGuidByPositionE6old(latE6, lngE6);
+
   // PLUGIN: cache > cache
   if (guid == null) {
     const cache = window.plugin.cachePortalDetailsOnMap;
     if (cache && cache.cache) {
       for (let g in cache.cache) {
         let p = cache.cache[g];
         if (latE6 == p.ent[2][2] && lngE6 == p.ent[2][3]) {
-          console.log("findPortalGuidByCacheE6 matched in cache! %s %s -> [%s] = %o", latE6, lngE6, g, p);
+//        console.log("newFindPortalGuidByPositionE6 matched in cache! %s %s -> [%s] = %o", latE6, lngE6, g, p);
           guid = g;
           break;
         }
       }
     }
   }
   return guid;
-}
+};
+
+window.cachePortalDetailsOnMap.searchInit = function () {
+    window.cachePortalDetailsOnMap.findPortalGuidByPositionE6old = window.findPortalGuidByPositionE6;
+    window.findPortalGuidByPositionE6 = window.cachePortalDetailsOnMap.findPortalGuidByCacheE6;
+};
 
-window.plugin.cachePortalDetailsOnMap.searchInit = function () {
-    window.plugin.cachePortalDetailsOnMap.findPortalGuidByPositionE6old = window.findPortalGuidByPositionE6;
-    window.findPortalGuidByPositionE6 = window.plugin.cachePortalDetailsOnMap.findPortalGuidByCacheE6;
-    console.log ("findPortalGuidByCacheE6 code injected");
-}
-/****************************************************************************************/
 window.plugin.cachePortalDetailsOnMap.setup  = function() {
 
   window.plugin.cachePortalDetailsOnMap.cache = {};
-  window.plugin.cachePortalDetailsOnMap.searchInit ();
+  window.plugin.cachePortaldetailsOnMap.searchInit;
```

Comment:
Seriously?

Reply:
Certainly not. Honestly I do not know how that got lost, my local backup has that brackets. 
Apart from that  I have no idea why this commit is in this PR. It has already been merged

Reply:
After an hour of debugging with @McBen (thank you) we have likely identified the cause.
The file was added two different branches and submitted in two PRs.
_This_ PR has the corrected version (see commit "0.3.9beta1" ad88cca).
It seems, that after the first PR was merged the file has a more recent timestamp than the file in _this_ PR.

An other thing puzzles us: 
![grafik](https://user-images.githubusercontent.com/25123137/102551684-33aff480-40c0-11eb-8ab4-02618e38d710.png)
This PR tells there is only one file with changes, but down here it offers two to review.
also this file is marked as outdated:
![grafik](https://user-images.githubusercontent.com/25123137/102551891-8c7f8d00-40c0-11eb-9191-045b158471f1.png)

Whatever goes wrong here, but do not include "cache-portals-on-map.js" in the merge.

Reply:
the bug is on master - not here
https://github.com/IITC-CE/ingress-intel-total-conversion/blob/3102c5c5cc47f9fd2e7933e633a99ed22f7c4117/plugins/cache-portals-on-map.js#L62

Reply:
@modos189 
Perhaps that should be reverted https://github.com/IITC-CE/ingress-intel-total-conversion/pull/388/commits/412fac2959f86cd789941f7354c2d4ff5d26f81b